src/index.ts

```diff
@@ -78,74 +88,76 @@ function times(...nums: numType[]): number {
   checkBoundary(leftValue);
 
   return leftValue / Math.pow(10, baseNum);
-}
+});
 
 /**
- * 精确加法
+ * Accurate addition.
+ *
+ * @param nums The numbers to add
  */
-function plus(...nums: numType[]): number {
-  if (nums.length > 2) {
-    return iteratorOperation(nums, plus);
-  }
-
-  const [num1, num2] = nums;
+const plus = createOperation((num1, num2) => {
   // 取最大的小数位
   const baseNum = Math.pow(10, Math.max(digitLength(num1), digitLength(num2)));
   // 把小数都转为整数然后再计算
   return (times(num1, baseNum) + times(num2, baseNum)) / baseNum;
-}
+});
 
 /**
- * 精确减法
+ * Accurate subtraction.
+ *
+ * @param nums The numbers to subtract
  */
-function minus(...nums: numType[]): number {
-  if (nums.length > 2) {
-    return iteratorOperation(nums, minus);
-  }
-
-  const [num1, num2] = nums;
+const minus = createOperation((num1, num2) => {
   const baseNum = Math.pow(10, Math.max(digitLength(num1), digitLength(num2)));
   return (times(num1, baseNum) - times(num2, baseNum)) / baseNum;
-}
+});
 
 /**
- * 精确除法
+ * Accurate division.
+ *
+ * @param nums The numbers to divide
  */
-function divide(...nums: numType[]): number {
-  if (nums.length > 2) {
-    return iteratorOperation(nums, divide);
-  }
-
-  const [num1, num2] = nums;
+const divide = createOperation((num1, num2) => {
   const num1Changed = float2Fixed(num1);
   const num2Changed = float2Fixed(num2);
+
   checkBoundary(num1Changed);
   checkBoundary(num2Changed);
+
   // fix: 类似 10 ** -4 为 0.00009999999999999999，strip 修正
   return times(num1Changed / num2Changed, strip(Math.pow(10, digitLength(num2) - digitLength(num1))));
-}
+});
 
 /**
- * 四舍五入
+ * Accurate rounding method.
+ *
+ * @param num The number to round
+ * @param decimal An integer specifying the decimal digits
  */
-function round(num: numType, ratio: number): number {
-  const base = Math.pow(10, ratio);
+function round(num: NumberType, decimal: number): number {
```

Comment:
'decimal' better to be `exponent` according to MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow#syntax